vars/dockerComposeRun.groovy

```diff
@@ -0,0 +1,19 @@
+import DockerCompose
+
+def call(service, command, tag='latest')
+{
+    try {
+        sh DockerCompose
+            .command('run', ['docker-compose.yml', 'docker-compose.ci.yml'])
```

Comment:
we don't need the `-f` here?

Reply:
it's added by https://github.com/elifesciences/elife-jenkins-workflow-libs/blob/master/src/DockerCompose.groovy#L53-L55